src/devtools/client/debugger/src/reducers/breakpoints.ts

```diff
@@ -16,23 +18,80 @@ import type { Breakpoint, SourceLocation } from "./types";
 export type { Breakpoint } from "./types";
 
 type LocationWithoutColumn = Omit<Location, "column">;
+
+export enum AnalysisStatus {
+  // Happy path
+  Created = "Created",
+  LoadingPoints = "LoadingPoints",
+  PointsRetrieved = "PointsRetrieved",
+  LoadingResults = "LoadingResults",
+  Completed = "Completed",
```

Comment:
Also for posterity, I'm going to note that while this set of Status's make sense, it's not the only set of options. For instance, I could picture having separate statuses for things like "PointsFound" and "Completed" and "WasReleased" and "Errored", because those things are not all mutually exclusive. Anyways, that's a problem for future us - those folks really know what they are doing.